### PR TITLE
Update salaries information in line with pay award

### PR DESCRIPTION
**Pull request by gemmadallmandfe**
### Trello card
https://trello.com/c/EfeNlPLs

### Context
Updating salaries in line with the pay award from September 2024

### Pages to review
* https://get-into-teaching-app-review-4155.test.teacherservices.cloud/life-as-a-teacher/pay-and-benefits/teacher-pay

* https://get-into-teaching-app-review-4155.test.teacherservices.cloud/
* https://get-into-teaching-app-review-4155.test.teacherservices.cloud/life-as-a-teacher/pay-and-benefits/career-progression
* https://get-into-teaching-app-review-4155.test.teacherservices.cloud/landing/grow
* https://get-into-teaching-app-review-4155.test.teacherservices.cloud/landing/how-much-do-teachers-get-paid
* https://get-into-teaching-app-review-4155.test.teacherservices.cloud/landing/careers
* https://get-into-teaching-app-review-4155.test.teacherservices.cloud/landing/what-teaching-taught-me
* https://get-into-teaching-app-review-4155.test.teacherservices.cloud/non-uk-teachers/train-to-teach-in-england-as-an-international-student
* https://get-into-teaching-app-review-4155.test.teacherservices.cloud/train-to-be-a-teacher

Two rounded variables removed and replaced with full values

One variable replaced with a new variable relating to a different salary value

Clarity added re **after** 5 years

### CODE REVIEW

File: app/views/content/landing/careers/_content.html.erb [hunk 1]
```diff
@@ -39,7 +39,7 @@
       </p>
 
       <ul>
-      <li>most teachers get a pay rise every year, depending on your school. You could earn over <%= v :salaries_fiveyears_maxshortened %> within 5 years</li>
+      <li>most teachers get a pay rise every year, depending on your school. You could earn over <%= v :salaries_fiveyears_maxshortened %> after 5 years</li>
```
Comment (emmaachesongray) [inline app/views/content/landing/careers/_content.html.erb:42]: Figures all look fine, but I think the content looks strange to have two sentences on the bullet point. Normally either all bullet points should be sentences and be capitalised, or not at all. 

```suggestion
      <li>most teachers get a pay rise every year (depending on your school), so you could earn over <%= v :salaries_fiveyears_maxshortened %> after 5 years</li>
```

File: app/views/content/landing/grow/_content.html.erb [hunk 2]
```diff
@@ -56,7 +56,7 @@
       </p>
 
       <ul>
-      <li>your salary is reviewed annually - most teachers get a pay rise every year, depending on your school. You could earn over <%= v :salaries_fiveyears_maxshortened %> within 5 years</li>
+      <li>your salary is reviewed annually - most teachers get a pay rise every year, depending on your school. You could earn over <%= v :salaries_fiveyears_maxshortened %> after 5 years</li>
```
Comment (emmaachesongray) [inline app/views/content/landing/grow/_content.html.erb:59]: Figures all look fine, but I think the content looks strange to have two sentences on the bullet point. Normally either all bullet points should be full sentences and be capitalised, or not at all. 
```suggestion
      <li>your salary is reviewed annually and most teachers get a pay rise (depending on your school), so you could earn over <%= v :salaries_fiveyears_maxshortened %> after 5 years</li>
```

File: app/views/content/landing/what-teaching-taught-me/_content.html.erb [hunk 3]
```diff
@@ -49,7 +49,7 @@
       </p>
 
       <ul>
-        <li>your salary is reviewed annually - most teachers get a pay rise every year, depending on your school. You could earn over <%= v :salaries_fiveyears_maxshortened %> within 5 years.</li>
+        <li>your salary is reviewed annually - most teachers get a pay rise every year, depending on your school. You could earn over <%= v :salaries_fiveyears_maxshortened %> after 5 years.</li>
```
Comment (emmaachesongray) [inline app/views/content/landing/what-teaching-taught-me/_content.html.erb:52]: Figures all look fine, but I think the content looks strange to have two sentences on the bullet point. Normally either all bullet points should be full sentences and be capitalised, or not at all. 
```suggestion
      <li>your salary is reviewed annually and most teachers get a pay rise (depending on your school), so you could earn over <%= v :salaries_fiveyears_maxshortened %> after 5 years</li>
```

File: config/values/salaries.yml [hunk 4]
```diff
@@ -1,60 +1,59 @@
-salaries_date_appliesfrom: 1 September 2023
+salaries_date_appliesfrom: 1 September 2024
 
 salaries:
 # Summary salaries
   starting:
```
Comment (emmaachesongray) [inline config/values/salaries.yml:5]: how do we decide whether to use shorthand or full figure?

File: config/values/salaries.yml [hunk 5]
```diff
@@ -1,60 +1,59 @@
...
-    minshortened: £30k
+    min: £31,650
+    minshortened: £31k
   fiveyears:
-    max: £41,333
-    maxrounded: £41,000
-    maxshortened: £41k
+    max: £43,607
+    maxshortened: £43k
   average:
```
Comment (emmaachesongray) [inline config/values/salaries.yml:11]: Couldn't find where these figures were on the doc?

**Review by emmaachesongray — COMMENTED** [review]
Couldn't find 2 of the figures on the doc, but all the others look good. Also had a couple of ad-hoc comments on the bullet points but feel free to ignore if not a priority :)

File: app/views/content/landing/grow/_content.html.erb (hunk 2 above)
Comment (gemmadallmandfe) [reply app/views/content/landing/grow/_content.html.erb:59]: Added to separate ticket

File: app/views/content/landing/what-teaching-taught-me/_content.html.erb (hunk 3 above)
Comment (gemmadallmandfe) [reply app/views/content/landing/what-teaching-taught-me/_content.html.erb:52]: Added to separate ticket

**Review by emmaachesongray — APPROVED** [review]
Looks good to me :)

**Pull request merged**